examples/analog_hamiltonian_simulation/09_Noisy_quantum_evolution_of_Bell_states.ipynb

```diff
@@ -0,0 +1,1628 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.            for i in range(int(len(post_sequence)/2)):
</pre><p>This is a bit hard to read, not sure why we are iterating up to half of the post_sequence length.</p><p>But in the next line the index is doubled: <code>state = post_sequence[2*i:2*i+2]</code></p><p>I wonder if there is a way to re-write this function more neatly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>First, as Peter suggested, I have removed this function "get_counters_from_result", and part of it has been moved below as "aggregate_qpu_counters". Second, here I am trying to get the count from QPU, where multiple copies of a pair of atoms are run in parallel.</p><p> </p><p>As an example, given the following result from the QPU {"grggrgrr": 10, "gggrrggg": 2}, the first key really means I got two occurernce for the states "gr", "gg", "rg", "rr" respectively in this order. Combined with the second key, it means I have {"gg": 14, "gr": 12, "rg": 12, "rr": 10}. I added this example into the doc-string, hopefully it helps.</p><p> </p><p> </p>